configure.ac

```diff
@@ -21,23 +21,20 @@ AC_SUBST(PKG_CONFIG_EXTRA_PATH, "")
 AC_SUBST(EXTRA_LD_FLAGS, "")
 
 AC_MSG_CHECKING([operating system])
-case "$host" in
-*-linux*)
-	AC_MSG_RESULT([$host (Linux)])
-	hidapi="hidapi-libusb"
-
-	#link with realtime lib on linux for clock_gettime
-	AC_SUBST(EXTRA_LD_FLAGS, "-lrt -lpthread")
-	;;
-*-freebsd*)
-	AC_SUBST(PKG_CONFIG_EXTRA_PATH, "libdata/")
-	;;
-esac
+AS_CASE(["$host"],
+	[*-linux*],
+		[AC_MSG_RESULT([$host (Linux)])
+		hidapi="hidapi-libusb"
+
+		#link with realtime lib on linux for clock_gettime
+		AC_SUBST(EXTRA_LD_FLAGS, "-lrt -lpthread")],
+	[*-freebsd*],
+		[AC_SUBST(PKG_CONFIG_EXTRA_PATH, "libdata/")])
 
 # Oculus Rift Driver
 AC_ARG_ENABLE([driver-oculus-rift],
-        [AS_HELP_STRING([--enable-driver-oculus-rift],
-                [enable building of Oculus Rift driver (default y)])],
+        [AS_HELP_STRING([--disable-driver-oculus-rift],
+                [disable building of Oculus Rift driver [default=yes]])],
```

Comment:
Yes, the entire help string is only there to tell the user what to type to disable, it doesn’t affect the behaviour. That is, you can still use --disable-driver-oculus-rift and --enable-driver-oculus-rift=no to disable it, and --enable-driver-oculus-rift or --enable-driver-oculus-rift=yes to force-enable it (which makes sense as a packager if you absolutely need it and there is a chance that upstream may change the default to no in the future, or has done that in the past).